src/Stryker.Core/Stryker.Core/StrykerRunResult.cs

```diff
@@ -15,6 +15,11 @@ public StrykerRunResult(StrykerOptions options, decimal? mutationScore)
 
         public bool IsScoreAboveThresholdBreak()
         {
+            if (MutationScore == null)
+            {
+                return _options.Thresholds.Break == 0;
```

Comment:
Why do you validate the breaking treshold here? If no mutations are made and my treshold > 0 I would get an exit code 1 right? Is this expected behaviour? 

I guess when no mutations are made you don't have a mutation score. Thus your score is neither lower or higher than the breaking treshold. I would expect a 0 exit code. @Mobrockers what do you think about this matter?

Reply:
> Why do you validate the breaking treshold here? If no mutations are made and my treshold > 0 I would get an exit code 1 right? Is this expected behaviour?

Oh, I didn't notice this scenario. That's true in this case it would return exit code 1. 
In that case 
```
if (MutationScore == null)
{
    return true;
}
```
should do the job. I think that if no tests were run, then we should have success message.

Reply:
Yes, this would be better for me! If you could update the PR it can be merged.